AElf.Management.Tests/SideChainServiceTests.cs

```diff
@@ -25,15 +26,17 @@ public async Task DeployTest()
             arg.LauncherArg = new DeployLauncherArg();
             arg.LauncherArg.IsConsensusInfoGenerator = true;
 
-            var service = new SideChainService();
+            var options = Options.Create(new ManagementOptions());
+            var service = new SideChainService(options);
```

Comment:
if unit tests use new operation, you should make it use IOC container.
if you have any problems, you can leave this module in static configuration, and make other static configuration convert to Options APSP.